woocommerce/woocommerce
# fix: CYS - change heading color

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Even if this PR should work, I didn't still get _why_ these changes are necessary :/. 

Maybe @chihsuan has a better context.

I don't understand why:
- we have to reset the core/heading color. TT4 doesn't have any customization for the core/heading block. ([theme.json](https://github.com/WordPress/twentytwentyfour/blob/trunk/theme.json))
- without this fix, the heading color is `--wp--preset--color--contrast`. The TT4 does not define this variable. Why does it happen? 

 

Closes #42940.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure you have WooCommerce Beta Tester installed.
2. Enable the customize-store feature flag in Tools -> WCA Test Helper -> Features.
3. Go to WooCommerce -> Home and click on Customize your store task.
4. Set up the store.
5. In the sidebar, click on `change the color palette`.
6. In the section `create your own palette`, update the heading color.
7. Ensure that the heading elements have the updated color.
8. Test the predefined palette.
9. Ensure that the heading elements have the updated color.


### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Fix: CYS - change heading color.
#### Comment 

</details>
